src/SortPeopleInDescendingOrder.java

```diff
@@ -0,0 +1,11 @@
+import java.util.Comparator;
+import java.util.List;
+
+public class SortPeopleInDescendingOrder {
```

Comment:
Is there any reason for creating a new class for this?

It may make more sense to create a new method in `SortPeopleByDateOfBirth` instead. This keeps all of the "sorting" functionality within one class instead of spreading it out